src/ptests/benchmark_hdf5.c

```diff
@@ -131,9 +139,18 @@ double t0, t1, t2;
  */
 double xtiming[15], timing[15], timingMin[15], timingMax[15];
 
+#define CD_VALUES_MAX 10
+size_t cd_nelmts=CD_VALUES_MAX;
+unsigned int cd_values[CD_VALUES_MAX];
+//hdf5_compress[0] = '\0';
+
+#ifdef H5Z_ZFP_USE_PLUGIN
```

Comment:
where is supposed to be defined H5Z_ZFP_PLUGIN
Does it come from H5pubconf.h ?

Reply:
No, ZFP is an external filter.  It would be defined in the compiler flags, -DH5Z_ZFP_USE_PLUGIN, when the ZFP filter is available. Currently, I don't test this option in the CI tests, manually only.